_studio/mfx_lib/shared/include/mfx_h264_encode_vaapi.h

```diff
@@ -295,6 +295,7 @@ namespace MfxHwH264Encode
         mfxU32 m_userMaxFrameSize;  // current MaxFrameSize from user.
         mfxU32 m_mbbrc;
         ENCODE_CAPS m_caps;
+        eMFXHWType m_hwType;
```

Comment:
Please, look into the comment below. If you will address it, you may not need to store hwType here in the class. Though if you will use it in some other place (another patch?), you may...